CODE_OF_CONDUCT.md

```diff
@@ -29,17 +26,22 @@ Examples of unacceptable behavior include:
 * Other behaviour which could reasonably be considered inappropriate in a
   professional setting
 
+We recognize that sometimes people may have a bad day, or be unaware of
+the impact of their behaviour. When that happens, you may carefully remind 
+them in public or private, whatever is more appropriate. Assume good faith;
+it's more likely that participants are unaware than that they intentionally
+try to denigrade others or the quality of the discussion.
+
 ## Maintainers and Responsibilities
 
 Project maintainers are those with commit rights to the Autocrypt specification.
-Each maintainer is asked to take responsibility and appropriate action in response
-to witnessed instances of unacceptable behavior.
-
+Each maintainer is asked to take responsibility and appropriate, careful action 
+in response to witnessed instances of questionable behavior.
 Project maintainers have the right and responsibility to remove, edit, or
 reject comments, commits, code, wiki edits, issues, and other contributions
-that are not aligned to this Code of Conduct, or to ban temporarily or
-permanently any contributor for other behaviors that they deem inappropriate,
-threatening, offensive, or harmful.
+that violate "Our Standards" above. 
```

Comment:
I liked the "are not aligned to this Code of Conduct" (without the rest of the sentence)

Reply:
k, reverted it.